osc-server-bom/root/opt/vmidc/bin/scripts/networkSettings.sh

```diff
@@ -14,32 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#Log current settings
-ip -o addr > log/networkSettings.log
-
-#Remove all network settings
-ip addr flush dev eth0 >> log/networkSettings.log
-
-if [ $1 = dhcp ]; then
-
-	sed -i -e '/BOOTPROTO/d' /etc/sysconfig/network-scripts/ifcfg-eth0
-	sed -i -e "$ a\\BOOTPROTO=dhcp" /etc/sysconfig/network-scripts/ifcfg-eth0
-	sed -i -e '/GATEWAY/d' /etc/sysconfig/network
-else
-
-	sed -i -e '/IPADDR/d' -e '/NETMASK/d' -e '/BOOTPROTO/d' /etc/sysconfig/network-scripts/ifcfg-eth0
-	sed -i -e '/GATEWAY/d' /etc/sysconfig/network
-	sed -i -e "$ a\\IPADDR=$2\\
-NETMASK=$3\\
-BOOTPROTO=static" /etc/sysconfig/network-scripts/ifcfg-eth0
-	sed -i -e "$ a\\GATEWAY=$4" /etc/sysconfig/network
-	rm -rf /etc/resolv.conf
-	echo "nameserver $5
-nameserver $6" > /etc/resolv.conf
-fi
-
-#Restart network
-/etc/init.d/network restart >> log/networkSettings.log 2>&1
-
-#Log settings after change
-ip -o addr >> log/networkSettings.log
\ No newline at end of file
+#Finds the Default Gateway from the route
+defaultGateway=`route -nee|egrep -e "^0.0.0.0"|tr -s " " |cut -d" " -f2`
```

Comment:
can you check what minimal "NAT- tools" instead of full "NAT-tool" installation in OSC container.

Reply:
is there any other way of getting the Gateway information?